### ISSUE TITLE
tfenv install latest:^0.11 fails

### ISSUE THREAD

**Issue by 4x0v7** [tfutils/tfenv#141]
```
$ tfenv --version
tfenv 1.0.1-8-gc3f5d5e
$ tfenv install latest:^0.11
[INFO] Installing Terraform v0.11.15
[INFO] Downloading release tarball from https://releases.hashicorp.com/terraform/0.11.15/terraform_0.11.15_linux_amd64.zip
#=#=-  #       #                                                                                                                                                                                                                                      
curl: (22) The requested URL returned error: 403 
tfenv: tfenv-install: [ERROR] Tarball download failed
```

Looks similar in spirit to [this issue](https://github.com/CodeKaio/gaia/issues/90)

**Comment by retornam** [comment]
fixed in https://github.com/tfutils/tfenv/pull/143

**Comment by Zordrak** [comment]
See #139 and #143

**Comment by Zordrak** [comment]
Being resolved #145
